### ISSUE TITLE
Bug: 7.2.2 Azure Service Bus: Another handler has already been assigned to this event

### ISSUE THREAD

**Issue by starkguerraj** [dotnetcore/CAP#1427]
Receiving the following exception after updating from 7.2.1 to 7.2.2

Using very simple implementation with Azure Service Bus, mostly all default settings with the exception of
UseDispatchingPerGroup = true

Example of two simple handlers with the following attributes
[CapSubscribe("Event1", Group = "Group1")]

[CapSubscribe("Event1", Group = "Group2")]

On start receiving the following exception.

Downgrading to 7.2.1 resolves the issue.

```
System.NotSupportedException: Another handler has already been assigned to this event and there can be only one.                                                                                 
    at Azure.Messaging.ServiceBus.ServiceBusProcessor.add_ProcessMessageAsync(Func`2 value)                                                                                                       
   at DotNetCore.CAP.AzureServiceBus.ServiceBusProcessorFacade.add_ProcessMessageAsync(Func`2 value)                                                                                                at DotNetCore.CAP.AzureServiceBus.AzureServiceBusConsumerClient.Listening(TimeSpan timeout, CancellationToken cancellationToken)                                                              
    at DotNetCore.CAP.Internal.ConsumerRegister.<>c__DisplayClass21_1.<Execute>b__2()            
```

**Comment by mstDe3** [comment]
Same issue. Seems to be it has been fixed as part of next PR.

[https://github.com/dotnetcore/CAP/pull/1426](https://github.com/dotnetcore/CAP/pull/1426)

**Comment by yang-xiaodong** [comment]
Fixed in `7.2.3-preview-216527974`
